### PR TITLE
fix(challenge): rename challenge, change fields to properties, fix test

### LINKED ISSUE #36325: ES6: Write Concise Object Literal Declarations Using Simple Fields
const createPerson = (name, age, gender) => {
  "use strict";
  // change code below this line
 1+3
  // change code above this line
};
console.log(createPerson("Zodiac Hasbro", 56, "male")); // returns a proper object

and this code works perfectly on the exercise.even its not the right solution

### PR DESCRIPTION
**Pull request by lasjorg**
1. The current challenge (on master) lets you pass using an empty object.

2. Rename the challenge to:

> Write Concise Object Literal Declarations Using Object Property Shorthand

3. Switch out the term fields for properties.

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/36325

### CODE REVIEW

**Review by thecodingaviator — COMMENTED** [review]
I think you'll need to change the name in `meta.json` too

**Comment by lasjorg** [comment]
@thecodingaviator whoops, thanks for that.

**Review by raisedadead — APPROVED** [review]
Platform code LGTM. Approved for QA and merge.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

For future reference, if you want you can use `DeepEqual` instead of comparing stringified objects.  It just does that for you.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-object-literal-declarations-using-object-property-shorthand.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ Use simple fields with object literals to create and return a <code>Person</code
 ```yml
 tests:
   - text: 'The output is <code>{name: "Zodiac Hasbro", age: 56, gender: "male"}</code>.'
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-object-literal-declarations-using-object-property-shorthand.md]: If we are going to fix this challenge, let's fix the `text` of the tests.
```suggestion
  - text: '<codecreatePerson("Zodiac Hasbro", 56, "male")</code> should return <code>{name: "Zodiac Hasbro", age: 56, gender: "male"}</code>.'
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-object-literal-declarations-using-object-property-shorthand.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ Use simple fields with object literals to create and return a <code>Person</code
...
-    testString: 'assert((() => {const res={name:"Zodiac Hasbro",age:56,gender:"male"}; const person=createPerson("Zodiac Hasbro", 56, "male"); return Object.keys(person).every(k => person[k] === res[k]);})(), ''The output is <code>{name: "Zodiac Hasbro", age: 56, gender: "male"}</code>.'');'
+    testString: assert((() => {const res={name:"Zodiac Hasbro",age:56,gender:"male"}; const person=createPerson("Zodiac Hasbro", 56, "male"); return Boolean(person) && JSON.stringify(res) === JSON.stringify(person);})());
   - text: No <code>key:value</code> were used.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-object-literal-declarations-using-object-property-shorthand.md]:
```suggestion
  - text: Your code should not use <code>key:value</code>.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my requested changes.

**Comment by lasjorg** [comment]
Sorry, I forgot to update the second assert message.

---

I do have a question. The challenge text says...

> return a `Person` object

...but it's not really a `Person` object, right? It's just an object.

Putting `Person` with capitalized P and having it inside a `<code>` tag makes it look (to me anyway) like it would be an object returned from a Person constructor invoked with the `new` keyword. But this is just a normal function returning an object. Shouldn't we remove `<code>Person</code>` from the text?

> Use object property shorthand with object literals to create and return an object with `name`, `age` and `gender` properties.

**Comment by RandellDawson** [comment]
@lasjorg I think your latest suggestion is a good idea.  Go ahead and commit it.

**Review by RandellDawson — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
@lasjorg Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :memo:

**Pull request merged**